### PR TITLE
chore(docs): Updated with Windows Configuration

### PR DESCRIPTION
**Pull request by JoseSoteloCohen**
Added SET GATSBY_GRAPHQL_IDE=playground&gatsby develop as the configuration needed to make it work in Windows.

## Description

Added SET GATSBY_GRAPHQL_IDE=playground&gatsby develop as the configuration needed to make it work in Windows.

After reading the docs I was unable to find a solution to why it wasn't working on my local environment until I finally found the solution on a Spectrum post, so I think that this information should be in the docs to make it easy for newcomers like myself.

### Documentation

[Gatsby Graphql Playground docs](https://www.gatsbyjs.org/docs/using-graphql-playground/)

### CODE REVIEW

**Comment by vladar** [comment]
We mention `cross-env` in one of our [articles](https://www.gatsbyjs.org/docs/scaling-issues/#gatsby_db_nodes). I'd recommend using it instead of `SET variable`

**Comment by JoseSoteloCohen** [comment]
> We mention `cross-env` in one of our [articles](https://www.gatsbyjs.org/docs/scaling-issues/#gatsby_db_nodes). I'd recommend using it instead of `SET variable`

Updated it! plus added a way to install cross-env

File: docs/docs/using-graphql-playground.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ To access this experimental feature utilizing GraphQL Playground with Gatsby, ad
 "develop": "GATSBY_GRAPHQL_IDE=playground gatsby develop",
 ```
 
+If you are on **Windows** then you should use:
+
+```
+"develop": "cross-env GATSBY_GRAPHQL_IDE=playground&gatsby develop",
```
Comment (vladar) [inline docs/docs/using-graphql-playground.md]: I don't think we need `&` with cross-env. So probably should be:

```suggestion
"develop": "cross-env GATSBY_GRAPHQL_IDE=playground gatsby develop",
```

Unless I am missing something?

**Comment by laurieontech** [comment]
@JoseSoteloCohen would love to get this merged! Could you address Vlad's most recent comment and we can get this in?

**Comment by JoseSoteloCohen** [comment]
@laurieontech Vladar is right, it wasn't needed so I accepted the suggestion :)

**Comment by laurieontech** [comment]
Awesome! Let's get this in.

**Pull request merged**